README.md

```diff
@@ -1,3 +1,41 @@
-# step112-2020
+# Ripple
+
+Ripple is a web app that serves as a platform for:
+    - Community members to easily discover local businesses, find real-time updates on safety measures being taken in response to COVID-19, and learn about the faces behind each business
+    - Local businesses to share personal stories and relevant information about their work in the community while growing their customer base virtually.
+
+
+Why Ripple?
+    - Small businesses have taken a large financial hit due to COVID-19 
```

Comment:
Consider: Small businesses have been significantly impacted by COVID-19 financially

Reply:
I changed it because it doesnt say whether it's a positive or negative impact. I'm sure people can infer, but I'd like it to be clear. If you have any other suggestions, lmk!

Reply:
Changed to: Small businesses have experienced significant financial strain due to COVID-19.

Reply:
SG